### ISSUE TITLE
Unable to launch any plugins

### ISSUE THREAD

**Issue by SamRaven2** [Berserker66/omnitool#19]
So I'll begin with saying I'm not very familiar with Python at all, so it's likely an error on my part causing this issue.

Whenever I attempt to load a plugin via the menu, I get the following trace in the console window:
![pythonomnitooltrace](https://cloud.githubusercontent.com/assets/7895337/9154722/9d7a7d38-3e6c-11e5-957f-095813006f8a.PNG)

I have python installed in the C:/Python34 directory with the matching architecture (both 32-bit). Same error happens with 64 bit python+omnitool.

I'm running 64-bit Windows 10 (that could be the issue too, who knows)
python version 3.4.3
Omnitool version 18.2.0

Any insight would be appreciated, as I would really like to use the program

**Comment by Berserker66** [comment]
so generic plugin interface is a bit broken right now. Any plugin that needs to read worlds doesn't work anyway, as anything past tile data isn't updated to 1.3 yet.
What does work is "Program" and "Generator" plugins, so those that only write or do not use world data.
What exactly are you trying to do? Is there a particular plugin you want to use?
Once #18 is merged, the plugingui should work again, so header reader and tombstone counter will work again, at least.

**Comment by SamRaven2** [comment]
I don't have a specific plugin that I was looking to use. I just noticed that this particular issue was not reported so I figured I would report it myself just in case. 

As long as it's a known issue and a fix is on the way (eventually) I can wait until then. The main thing I was looking to check that it wasn't just me messing something up.

Thank you for the time and work you put into the program.
